rllib/agents/trainer.py

```diff
@@ -535,8 +536,14 @@ def _setup(self, config):
         env = self._env_id
         if env:
             config["env"] = env
+            # An already registered env.
             if _global_registry.contains(ENV_CREATOR, env):
                 self.env_creator = _global_registry.get(ENV_CREATOR, env)
+            # A class specifier.
+            elif "." in env:
```

Comment:
This is kind of cool but can you please split it out into a separate PR?

Reply:
done